docs/input/docs/reference/requirements.md

```diff
@@ -28,11 +28,6 @@ The repository needs to have an existing local `master` or `main` branch.
 For some branch strategies (such as [Git Flow][gitflow]), a local `develop`
 branch needs to exist.
 
-### Switched branch
-
-The repository should be [switched][git-switch] to a named, existing branch
-pointing to the commit being built (i.e. no detached `HEAD`).
-
```

Comment:
I agree this section should be removed, but can you please add another section somewhere relevant that explains the `no-branch` naming as a result of a detached `HEAD`?

Reply:
I added the documentation to [docs/input/docs/reference/version-increments.md](https://github.com/U7nk/GitVersion/commit/00b845869f695b6e83fb89a2b82d3046c8fd0381#diff-c66dc52bd6d65e74405780ba69d21324099c18b9488757e36111ee4a49977508).
 I'm not sure if this is the right place.

Reply:
Perfect!